### PR TITLE
Make spacer growable in order to align control elements

### PR DESCRIPTION
**Pull request by tomasz-oponowicz**
## Test case

1. Open the test page
1. Load the Angel asset
1. Enter full screen mode

### Expected

The current time element is left-aligned and other controls are right-aligned.

### Actual

The current time element isn't left-aligned but has an indentation. For example:

before

### CODE REVIEW

**Comment by tomasz-oponowicz** [comment]
I signed it!

File: ui/less/other_elements.less [hunk 1]
```diff
@@ -35,11 +35,10 @@
...
 }
 
-/* This is a spacer element used to separate elements within the contol
+/* This is a spacer element used to separate elements within the control
  * buttons panel. It's just an empty div of certain width. */
 .shaka-spacer {
-  width: 48%;
 
-  /* Make the element shrink to accommodate things to the right. */
-  .shrinkable();
```
[the remark this answers is not in the document]
Comment (tomasz-oponowicz) [reply ui/less/other_elements.less]: @ismena Thanks for your feedback. 

I applied requested change.

Additionally I removed margins in order to make spacer more flexible when shrinking (margins don't shrink).

**Comment by ismena** [comment]
Looks good! Let me run the tests.

**Comment by ismena** [comment]
Merged! Thanks for the contribution!

**Pull request merged**